telos-frontend/src/components/habit-tracker/JournalHabitTracker.jsx

```diff
@@ -1,34 +1,313 @@
-import AddIcon from '@material-ui/icons/Add';
-import IconButton from '@material-ui/core/IconButton';
-import './JournalHabitTracker.css';
-
-const DashboardHabitTracker = () => (
-  <div className="container">
-    <div className="HabitualsTitle">
-      <h2 align="middle">Habituals</h2>
-      <span className="AddIcon">
-        <IconButton>
-          <AddIcon />
-        </IconButton>
-      </span>
-    </div>
-    <div className="Habituals">
-      <ul>
-        <div>
-          <li>H1</li>
-          <p>Habit Name</p>
-        </div>
-        <div>
-          <li>H2</li>
-          <p>Habit Name</p>
-        </div>
-        <div>
-          <li>H3</li>
-          <p>Habit Name</p>
-        </div>
-      </ul>
-    </div>
-  </div>
-);
-
-export default DashboardHabitTracker;
+import AddIcon from '@material-ui/icons/Add';
+import IconButton from '@material-ui/core/IconButton';

```

Comment:
These imports could be combined into one import from @material-ui/core

Reply:
done